_posts/2022-02-01-OLS-4-cohort-experience.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: "OLS-4 cohort experience"
+image: https://github.com/open-life-science/open-life-science.github.io/blob/main/images/logo.png
+authors:
+   - Arent Kievits
+   - Open Science Community Delft
+---
+
+*A graduate of the fourth cohort of the Open Life Science (OLS) program talks about his experience!*
+
+As a PhD student at Delft University of Technology, I participated in the fourth cohort (named *Kenaz*) of the Open Life Science (OLS) program. OLS is a 16-week program where participants receive personal mentoring and training through cohort-calls. The main objectives are to:
+
+- Gain knowledge about open science practices from experts
+- Share expertise and learn to create, lead and sustain an Open Science project
+- Connect with OLS community members from different backgrounds, identities and work together on projects
+- Become a contributing member of OLS and ambassador of open science
+
+# My motivation for starting an open science project
+I started as a science rookie in november 2020, embarking on a PhD journey where I was going investigate how to image biological samples in 3D with a new type of electron microscope. I wanted to kickstart my science career with a firm background in open science and publish and share my work in an open way. Being able to tie my OLS training to my own PhD motivated me to initiate an open science project, [Open Multibeam Scanning Electron Microscopy (OpenMB-SEM)](https://github.com/arentkievits/OpenMB-SEM). The ideas behind this project are: 
+
+- **No restrictions on access**: If people want to validate the work, they should have all resources and instructions to do so.
+- **Solving challenges together**: We have a way more powerful and faster electron microscope than previously possible, which stresses existing and creates additional challenges.
+- **Benefit to the EM community**: As we are one of the first to have such a microscope, we want others to benefit as well!
+- **Early input on the design process**: We are pioneering this technique. We could use help from experts in the field to help us write software, annotate data, think about sample preparation, etc.
+
+# My goals and expectations
+I set out some goals at the beginning of the OLS-programme. I did not know how the project was going to look like at that moment. Luckily, my assigned mentor contacted an expert in my field to help co-mentor me. Together we consolidated the goals: 
+
+- Sharing research results in a way that others can effectively re-use them
+- Start using GitHub for sharing software (Python).
+- Improve documentation for my code
+- Publish open access
+- Extend our electron microscopy data viewer to include more info on the samples, types of experiments, etc.
+- Build a website with methods, code, data and literature for multibeam electron microscopy (MB-SEM)
+
+And suggested by my mentor:
+- Consider re-activating Twitter (though honestly, I fail miserably at maintaing my socials...)
```

Comment:
maintai**n**ing (and agree completely, social media is hard.)

Reply:
> This is awesome @arentkievits! Very well written and great to see the progress you've made with your mentors 💯 I've made some small comments :) Also - could we please crosspost this on the [OLS blog](https://openlifesci.org/posts)? I can arrange that :)

Thanks! You can crosspost it to the OLS blog :)